core/src/main/java/co/aikar/commands/contexts/OptionalContextResolver.java

```diff
@@ -27,6 +27,12 @@
 import co.aikar.commands.CommandIssuer;
 
 /**
- * Context Resolver that can accept null input
+ * The same as {@link ContextResolver}, however it can accept a null context.
```

Comment:
append, if the parameter was marked optional, will still be called with an empty args list